DiceKitTests/FrequencyDistributionIndex_Tests.swift

```diff
@@ -35,7 +45,65 @@ extension FrequencyDistributionIndex_Tests {
 // MARK: - Equatable
 extension FrequencyDistributionIndex_Tests {
     
-    // TODO: Equatable
+    func test_shouldBeReflexive() {
+        property("reflexive") <- forAll {
+            (a: SwiftCheckOrderedOutcome) in
+            
+            let a = a.getSet
+            
+            let index = Int(arc4random_uniform(UInt32(a.count)))
```

Comment:
I wish there was a way to generate a random number within a range. I guess at a minimum I should extract this to a utility function.